### PR TITLE
better default tinymce settings

### PR DESCRIPTION
**Pull request by vangheem**

### CODE REVIEW

**Comment by vangheem** [comment]
test running: http://jenkins.plone.org/job/pull-request-5.0/297/

File: Products/CMFPlone/interfaces/controlpanel.py [hunk 1]
```diff
@@ -547,10 +548,9 @@ class ITinyMCEPluginSchema(Interface):
             SimpleTerm('visualchars', 'visualchars', u"visualchars"),
             SimpleTerm('wordcount', 'wordcount', u"wordcount")
         ])),
-        default=['advlist', 'directionality', 'emoticons',
-                 'fullscreen', 'hr', 'insertdatetime', 'lists', 'media',
+        default=['advlist', 'fullscreen', 'hr', 'lists', 'media',
                  'nonbreaking', 'noneditable', 'pagebreak', 'paste', 'preview',
-                 'print', 'save', 'searchreplace', 'tabfocus', 'table',
+                 'print', 'save', 'tabfocus', 'table',
```
Comment (thet) [inline Products/CMFPlone/interfaces/controlpanel.py]: did you remove ``searchreplace`` instead of ``save``?

Comment (vangheem) [inline Products/CMFPlone/interfaces/controlpanel.py]: No, this was on purpose. There is no integration with "save" so it was removed. See note above new "save" button.

Comment (thet) [inline Products/CMFPlone/interfaces/controlpanel.py]: I know you removed the ``save`` option on purpose from the vocabulary. but you did not remove it from the ``default`` list. instead you removed ``searchreplace``.

Comment (vangheem) [inline Products/CMFPlone/interfaces/controlpanel.py]: bah, I can't read. I see now.

Comment (thet) [inline Products/CMFPlone/interfaces/controlpanel.py]: np :)

**Pull request merged**